app/backend/src/clj/teet/meeting/meeting_commands.clj

```diff
@@ -240,43 +240,26 @@
           :type :notification.type/meeting-updated
           :project project-eid}))))))
 
-(defn email-subject-for-type [meeting type]
-  (case type
-    :invitation
-    (str "TEET: koosoleku kutse: " (meeting-model/meeting-title meeting) " / "
-         "TEET: meeting invitation: " (meeting-model/meeting-title meeting))
+(defn email-subject [meeting]
+  (str "TEET: koosolek uuendatud: " (meeting-model/meeting-title meeting) " / "
```

Comment:
move translations to translation files, so we don't need code changes if they need to be updated

Reply:
Did we have some beatiful way of using translations in the backend?

Reply:
yes, we have the same style in backend as well